internal/controller/operatorconfigmap_controller.go

```diff
@@ -469,6 +481,45 @@ func (c *OperatorConfigMapReconciler) reconcileDelegatedCSI() error {
 	return nil
 }
 
+func (c *OperatorConfigMapReconciler) getDomainLabels() ([]string, error) {
+	topologyLabels := []string{
+		"kubernetes.io/hostname",
+		"topology.kubernetes.io/zone",
+		"topology.rook.io/rack",
+	}
+	actualTopologyLabels := []string{}
+
+	// list node labels
+	nodes := &metav1.PartialObjectMetadataList{}
+	nodes.SetGroupVersionKind(corev1.SchemeGroupVersion.WithKind("NodeList"))
+	labelSelector := &client.ListOptions{
```

Comment:
@travisn @malayparida2000 @rewantsoni there can be any arbiter zone at client side??